### PR TITLE
[frontend] Update Vagrant box to 42.2

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: contrib/common.sh [hunk 1]
```diff
@@ -24,7 +24,6 @@ function install_common_packages() {
     ruby2.4-rubygem-bundler \
     ruby2.4-rubygem-mysql2 \
     ruby2.4-rubygem-nokogiri \
-    ruby2.4-rubygem-multi_json \
```
Comment (Ana06) [inline contrib/common.sh]: Is this really related to the vagrant box update?

Comment (hennevogel) [inline contrib/common.sh]: sure. we don't need to install this via packages...
